### DISCUSSION TITLE
Is there a way to disable/hide the plus button of 'upload file' and 'create bucket'

### DISCUSSION THREAD

**Discussion by Itzikbenza** [minio/minio#13110] [Q&A]
Hello, 
I try to avoid the option to upload a file or create an bucket through the button at the bottom of the screen.
(The red plus button)

Is there a way?

**Comment by harshavardhana** [comment]
If you only have read only permissions in your IAM policy - it should be possible for UI disable those buttons.

**Reply by Itzikbenza** [reply]
Thanks for your reply!  So how do I make the change of disabling the plus button from the UI?

**Reply by harshavardhana** [reply]
Not right now, we will have to work on it as a feature

**Reply by Itzikbenza** [reply]
So how can I change to read-only permissions?

**Reply by ravindk89** [reply]
Create a [policy](https://docs.min.io/minio/baremetal/security/minio-identity-management/policy-based-access-control.html) that is read only for the bucket and apply it to the user. We have canned policies you can use for doing this.

The linked documentation and its sibling pages have the information you need.
